centreon/src/Core/Application/Common/UseCase/AbstractPresenter.php

```diff
@@ -23,16 +23,14 @@
 namespace Core\Application\Common\UseCase;
 
 use Symfony\Component\HttpFoundation\Response;
-use Core\Application\Common\UseCase\PresenterInterface;
-use Core\Application\Common\UseCase\ResponseStatusInterface;
 use Core\Infrastructure\Common\Presenter\PresenterFormatterInterface;
 
 abstract class AbstractPresenter implements PresenterInterface
 {
     /**
      * @var ResponseStatusInterface|null
      */
-    protected $responseStatus;
+    protected ?ResponseStatusInterface $responseStatus = null;
```

Comment:
This property is now used only by this AbstractClass thanks to this refactor.
Why not set it **private** to avoid bad practice ?